Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1287,9 +1287,15 @@ - (void)_scrollToContentScrollPosition:(WebCore::FloatPoint)scrollPosition scrol
 
     [_scrollView _stopScrollingAndZoomingAnimations];
 
-    if (!CGPointEqualToPoint(contentOffsetInScrollViewCoordinates, [_scrollView contentOffset]))
+    if (!CGPointEqualToPoint(contentOffsetInScrollViewCoordinates, [_scrollView contentOffset])) {
```

Comment:
Maybe store `[_scrollView contentOffset]` in a local variable? It's used five times here.